draft-ietf-quic-qpack.md

```diff
@@ -952,10 +954,8 @@ entry.
 
 ### Indexed Header Field
 
-An indexed header field representation identifies an entry in either the static
-table or the dynamic table and causes that header field to be added to the
-decoded header list, as described in Section 3.2 of [RFC7541].
-<!-- is the 7541 reference still helpful here -->
+An indexed header field instruction identifies an entry in either the static
```

Comment:
Why did you change this from representation to instruction?  Though this isn't currently spelled out directly, the encoder/decoder streams contain "instructions" - which cause the receiver to take an action.  Header blocks contain representations, which emit headers fields.  The word instruction doesn't appear in this section.

The other representations don't seem to call themselves such, however.  And only this one contains the language about adding to the decoded header list.  I guess I prefer emit also, but maybe we need to make another consistency pass.

Reply:
Created issue #3133 as the "another consistency pass".